tests/integration/pilot/outboundtrafficpolicy/helper.go

```diff
@@ -283,20 +283,24 @@ func RunExternalRequestTest(expected map[string][]string, t *testing.T) {
 				portName string
 				host     string
 				gateway  bool
+				schema   scheme.Instance
 			}{
 				{
 					name:     "HTTP Traffic",
 					portName: "http",
+					schema:   scheme.HTTP,
 				},
 				{
 					name:     "HTTPS Traffic",
 					portName: "https",
+					schema:   scheme.TCP,
```

Comment:
Same here, better to add new ones and avoid changing the current test cases.

Reply:
The test cases failed. If the port protocol is HTTPS, the test framework will deploy TCP server but the client still forward HTTP request.

Reply:
I am confused. We need to send an HTTP(s) request, why are we now sending TCP? We can add another case for TCP but we can't remove the https case

Reply:
https://github.com/istio/istio/blob/ebf88012909956f29db99fc5f11d34f5b0723ad2/pkg/config/protocol/instance.go#L124

We treat HTTPS as TCP protocol

Reply:
So before, we were sending a request like `GET / HTTP1.1 ...` (with TLS). Now we are sending a request like `HelloWorld` (with TLS (?))? Is that right?

Reply:
Yes.